internal/stores/accounts.go

```diff
@@ -0,0 +1,38 @@
+package stores
+
+import (
+	"math/big"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/renterd/ephemeralaccounts"
+	rhpv3 "go.sia.tech/renterd/rhp/v3"
+)
+
+type (
+	dbEphemeralAccount struct {
+		Model
+
+		Owner string `gorm:"NOT NULL"`
+
+		// AccountID identifies an account. It's a public key.
+		AccountID rhpv3.Account `gorm:"unique;NOT NULL"`
+
+		// Host describes the host the account was created with.
+		Host types.PublicKey `gorm:"unique;NOT NULL"`
```

Comment:
I think there should be a combined unique index on host + account id

Reply:
I removed the `unique` on the host because that doesn't make sense anymore but I'll think a bit more about indices in the persistence PR that's going to follow. I think we might not even need one since we only fetch from that table once and we always fetch all accounts. We also shouldn't reuse account IDs ever. Because then you might end up with replays between hosts.